src/alto-ui/Tag/Tag.scss

```diff
@@ -0,0 +1,48 @@
+@import '../scss/inc';
+
+.Tag {
+  background-color: $blue-10;
+  border: 1px solid $blue-20;
+  color: $blue;
+  border-radius: $border-radius-default;
+  padding: $spacing-x-small $spacing-small;
+  display: flex;
+  align-items: center;
+  line-height: 1;
+
+  &--button {
+    @extend %reset-button;
+    transition: all $transition;
+    cursor: pointer;
+
+    &:focus,
+    &:hover {
+      border-color: $blue;
+    }
+  }
+
+  &--rounded {
+    border-radius: 0.5rem + $spacing-x-small;
+  }
+
+  &--button:active,
+  &--active {
+    background-color: $blue;
+    color: $white;
+  }
+
+  &--disabled {
+    &,
+    &.Tag__button {
+      &,
+      &:focus,
+      &:hover,
+      &.Tag--active {
```

Comment:
You said that we should avoid such nesting of `&` - hard to figure out which `&` means what.  Maybe write normal class to improve readability ?

Reply:
The idea was to reduce the number of reducer and lines,
do you prefer this:

```scss
&--disabled,
&--disabled.Tag__button
&--disabled:focus,
&--disabled.Tag__button:focus,
&--disabled:hover,
&--disabled.Tag__button:hover,
&--disabled.Tag--active,
&--disabled.Tag__button.Tag--active {
``` 

?

Reply:
👍 let`s keep as it is